### PR TITLE
Commit the initial gruntfile is created

### PR DESCRIPTION
**Pull request by AngelaPrades**
Creating the gruntfile with 2 tasks: uglify and web

The server is local and it is located at localhost:8080

Page for index is:
localhost:8080/src/index.html

right now

### CODE REVIEW

**Comment by AngelaPrades** [comment]
Ok, I'll remove it from the bower file

File: gruntfile.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            hostname: 'localhost',
+            port: 8080,
+            keepalive:true,
+            open: {
+              target: 'http://localhost:8080/src/index.html'
+            }            
+          },
+        }
+     },
+      uglify: {
```
Comment (arp82) [inline gruntfile.js]: Please, remove uglify for now.

**Comment by anenchu** [comment]
I think the code is good now, and it seems it works fine. In my opinion, go ahead with the merge.

**Comment by AngelaPrades** [comment]
Yesterday before finish the class I fixed all comments.

Merge is done according to the Ana comment.

**Pull request merged**